bucket/treesize-free.json

```
@@ -17,9 +17,9 @@
     ],
     "checkver": {
         "url": "https://www.jam-software.com/treesize_free/changes.shtml",
-        "regex": "Version\\s+([\\d.]+):"
+        "regex": "Version ([\\d.]+)\\</h3\\>"
```

Why do you escape anchor?

```suggestion
        "regex": "Version\\s+([\\d.]+)</h3>"
```